medical_compliance/medical_compliance/api/store_utils.py

```diff
@@ -115,7 +111,8 @@ def insert_measurement(store_endpoint_host_uri, user_id, device_id,
 
         "user": user_uri,
         "device": device_uri,
-        "value_info": value_info
+        "value_info": value_info,
+        "ok": True
```

Comment:
@cretualexandru: I see the utility of this field, but wouldn't it be better if we instead use a `severity` field that allows for more nuanced values -- ex: `normal`, 'low', `high` ? My idea is that based on the analysis we could still trigger notifications, but with different priorities and/or properties and means to interact with them.